src/Components/TicTacToe/TicTacToe.module.css

```diff
@@ -0,0 +1,17 @@
+.square {}
+
+.square-cross {
```

Comment:
When using CSS modules, it's recommended to write classnames in `camelCase`. Then you can simply use them with `styles.squareCross` instead of `styles["square-cross"]`